### ISSUE TITLE
Floating settings tab

### ISSUE THREAD

**Issue by tdechangy** [ThemeFuse/Brizy#773]
Globally settings seem to be don right up to now but it sadly overhanging the content which can be more than frustrating when working on the right side of the screen.

Ability to chose to display floating settings (as proposed under Divi) is pretty nice UX. In fact those settings could eventually be placed over the left settings bar WITHOUT" overhanging the content. All builders work like this and this is the way to do. Otherwise designers will get quickly frustrated, and that's certainly not the goal.

- I'm waiting also for serious responsive setups - i know it's in the roadmap.

- Just to repeat this point : WPML compatibility is a MUST for any European website! 
No WPML, No Brizy.

**Comment by DimiBaitanciuc** [comment]
Good point. We always knew this will pose some inconveniences when working on the right side of the page. We plan to fix that and we had something like you are suggesting in mind. 

Most likely this will get sorted out once we flesh out the product a bit more. The plan is to add all the main features we have on the list this year and then go back and start to polish everything to perfection.

**Comment by tdechangy** [comment]
Good to hear that you take this in serious consideration. You might also have a look at Oxygen which is quite clean. I also particularly appreciate their NO-theme approach since I hat the need to go through the "additional" the settings panel while it should be done quicker and smoother with an all-in-one builder. One down-side I realized, was the impossibility to add functions easily! yes, we have to build plugins for it, quite crazy when you like to use many PHP codes for Woocommerce for example, using a clever risk-free, plugin to implement php snippets.

Since i didn't have any answer to my pre-purchase questions through the website chat (messenger) , I would like to address some questions here if you allow me. Or by pm or through messenger @ Thibault de Changy.

To go global, I have at least these main questions in mind :
- What do you think will make you apart from the competition ?
- Will you ever stay related to themes or go the more global way as does Oxygen ? I would probably appreciate a free 100% compatible near empty (ajax) theme, with minimal settings for maximum speed and perfect SEO.
- WPML compatibility is a must for EU users, so I won't be able to use it until then. I need that you add a Trello card to be sure it's on your roadmap asap.

+ advanced features that could make a difference : 
- a visual CSS grid system as seen here : https://pinegrow.com/docs/css-grid/?highlight=grid It's awesome !
- Forms for all custom fields with image/media & file upload,  usable for membership & vendor sites.

Waiting to read you soon about the main features specifications to come.
